ADMIN.md

```diff
@@ -76,6 +76,21 @@ Add new library to "nightly" and "publish" CI [workflows](https://github.com/dfi
 
 ## Release
 
-To release a version to npm, create a release in GitHub. This will start an action that build and publish the libraries.
+Steps to release new versions of the libraries.
+
+1. Create a PR bumping versions
+
+- Upgrade `"version"` in all needed packages.
```

Comment:
Maybe something like following to be more accurate then:

```
- Upgrade `"version"` in the various library `packages.json`. If package depends on another package of ic-js, update the reference as well.
- Update `"version"` in the root `packages.json` of the project
- Run `npm i` from root directory
- Update Changelog
```

Reply:
done!